templates/auditbeat.yml.j2

```diff
@@ -83,7 +83,10 @@ output.elasticsearch:
   password: {{ auditbeat_output.elasticsearch.security.password }}
   protocol: {{ auditbeat_output.elasticsearch.security.protocol }}
 {% if auditbeat_output.elasticsearch.security.ssl_certificate_authorities is defined %}
```

Comment:
This should have only one if block, also I suggest adding default("full") to auditbeat_output_elasticsearch.ssl.verification_mode
```
{% if auditbeat_output.elasticsearch.security.ssl_certificate_authorities is defined %}
  ssl.certificate_authorities: {{ auditbeat_output.elasticsearch.ssl_certificate_authorities | to_json }}
  ssl.verification_mode: {{ auditbeat_output.elasticsearch.ssl.verification_mode | default("full") }}
{% endif %}
``